pnpm-lock.yaml

```diff
@@ -101,8 +101,13 @@ importers:
       '@fluentui/react-focus': 8.8.10
       '@fluentui/style-utilities': 8.8.3
       '@fluentui/utilities': 8.13.4
+<<<<<<< HEAD
```

Comment:
Could it be that some merge resolution left-overs got into this file

Reply:
yeah I did have multiple merge conflicts